SparkPost/2web2ui
# EO-611 Release Alerts UI - [Merge on 5/14]

## To be merged on 5/14/2019

### What Changed
- Routes
  - Moves all `/alerts-new` routes to `/alerts` 
  - Removes access control checks for the `feature-alerts` UI options
- Alerts pages
  - Moves all content under `src/pages/alerts-new` into `src/pages/alerts`
  - Removes the old alerts landing page
- Account nav
  - Removes the alerts nav item

### How To Test
- Log into an account without the `feature-alerts` UI option
- Verify alerts appears in the side nav
- Verify all links and redirect within the alerts UI use the correct `/alerts` path

### To Do
